integrations/github-files/src/github.ts

```diff
@@ -1,14 +1,17 @@
 import { ExposableError } from '@gitbook/runtime';
-import { GithubInstallationConfiguration, GithubRuntimeContext } from './types';
+import { GithubInstallationConfiguration, GithubRuntimeContext, GithubSnippetProps } from './types';
 
 export interface GithubProps {
     url: string;
 }
 
+const constructGithubUrl = (owner: string, repo: string, branch: string, filePath: string) => {
+    return `https://github.com/${owner}/${repo}/blob/${branch}/${filePath}`;
+};
+
```

Comment:
technically this won't work for self hosted github enterprise server instances, not sure if that matters. If so, we can change the config to include a baseline root URL